README.md

```diff
@@ -265,6 +265,9 @@ There are plans to support more Publisher properties but for now you will have t
 | session | [Session](https://tokbox.com/developer/sdks/js/reference/Session.html) | No | OpenTok Session instance. This is auto populated by wrapping `OTSubscriber` with `OTStreams`
 | stream | [Stream](https://tokbox.com/developer/sdks/js/reference/Stream.html) | No | OpenTok Stream instance. This is auto populated by wrapping `OTSubscriber` with `OTStreams`
 | properties | Object | No | Properties passed into `session.subscribe`
+| retry | Boolean | No | Set true to retry the subscribe process in case of failure (Default: false)
+| maxRetryAttempts | Number | No | Max retry attempts in case of subscribe failure (Default: 15)
```

Comment:
I think setting 15 as the `maxRetryAttempts` is a bit much. How about 5?

Reply:
Agree. Yes, 15 is too much 👍